docs/content/en/docs/operator-manual/operations/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Operations"
```

Comment:
How about placing these docs under the `/operator-manual/control-plane/` dir? I feel all of them are the information needed for the control-plane operator.

Reply:
I get your point 👍 I was thinking that way at first but in a second thought, I want to make a group contains kinds of guidelines or step-by-step operation tutorial (not only for the control-plane operator but also the piped management). How do you think about that 🤔

Reply:
I feel you. I got what you're wanting to do. But honestly, I would prefer to explicitly divide the operator manual into two sections, [Control-plane](https://pipecd.dev/docs/operator-manual/control-plane/) and [Piped](https://pipecd.dev/docs/operator-manual/piped/).

The reason why is because initially we kind of defined the three actors bondary:
- User
- Piped Operator
- Control-plane Operator

And, we wanted to make it so that control-plane operators only had to look at a certain page. (As well as Piped operators only had to look at a certain page)

>I want to make a group contains kinds of guidelines or step-by-step operation tutorial

I feel the same way. I also think we need more tutorials and to organize them in a single group. For tutorials that involve more than one area (including Piped and Control-plane), they should be placed in this new operations section. But I feel like this migration guide is slightly different from it.

But I can't say the current documentation structure is enough readable for all users. So some kinds of transformation are obviously needed. Let's wait for @nghialv 's thought!

Reply:
>  I also think we need more tutorials and to organize them in a single group. 

More than agree 😂 

> we wanted to make it so that control-plane operators only had to look at a certain page. 

You convinced me, I started thinking as my first thought already 🤔 But let's wait for some other idea 👍

Reply:
Yes, I think they should be at `/operator-manual/control-plane/operations` because those operations are just for control-plane operators.

Reply:
@nghialv Thank you for looking over them!

Reply:
@nghialv just to clarify, will we create `/operator-manual/control-plane/operations` directory and place this `migrate-datastore` post to that directory 👀

Reply:
Yes. The path of this file should be `docs/content/en/docs/operator-manual/operations/control-plane/datastore-migration.md`.